secant/UI Components/CheckCircle/CheckCircle.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  ZcashCheckCircle.swift
+//  wallet
+//
+//  Created by Francisco Gindre on 1/7/20.
+//  Copyright © 2020 Francisco Gindre. All rights reserved.
+//
+
+import SwiftUI
+
+struct CheckCircle: View {
+    @Binding var isChecked: Bool
```

Comment:
Shall we change this to be an action of a state and remove the binding for a more TCA oriented approach ?
@LukasKorba

Reply:
The `CheckCircle` should be ideally reimplemented so it's a TCA component with its own store and reducer. That is the purest form of composability and `CheckCircle` could be reused. 

If we're sure this is the only place in the app to ever use `CheckCircle` we could also reuse some of the existent Store (from the parent view) but it's not the ideal solution.

The bool lives in the State so store's `bindingFor<purpose>` needs to be implemented and passed to `CheckCircle(isChecked: store.bindingFor...).

```swift
// should be removed completely
@Binding var isChecked: Bool
```

Reply:
I didn't use the binding because it didn't make sense to me. But I think I made pure TCA component from it. Please let me know what do you think.